controllers/reef_intake.py

```diff
@@ -54,8 +54,12 @@ def intaking(self, initial_call: bool):
                 self.done()
                 return
 
+            if self.injector_component.has_algae():
+                self.done()
+
         if self.injector_component.has_algae():
             self.next_state("safing")
```

Comment:
I think this `next_state` call would override the above `done` call

Reply:
fixed